liota/lib/utilities/utility.py

```diff
@@ -314,33 +314,42 @@ def read_user_config(config_file_path):
     return user_config
 
 
-def validate_named_pipe(pipe_file):
+class DiscUtilities:
     """
-    Check whether a named pipe exists: if not, create it and set permission.
-    :param pipe_file: path of a named pipe file
-    :return: True or False (named pipe finally exists or not)
+    DiscUtilities is a wrapper of utility functions
     """
-    assert (isinstance(pipe_file, basestring))
-    if os.path.exists(pipe_file):
-        if stat.S_ISFIFO(os.stat(pipe_file).st_mode):
-            pass
+
+    def __init__(self):
+        pass
+
+    def validate_named_pipe(self, pipe_file):
+        """
```

Comment:
I can see comment section added which wasn't part of Winnie`s initial commit. Hence, this is not a revert commit.

Reply:
As discussed, this is a revert commit. utility.py and liotapkg.sh where having merge conflicts, and hence, these changes were manually merged.

A revert commit should only take care of reverting the changes, and should not introduce any new changes.